src/schannel_cred.rs

```diff
@@ -167,10 +169,24 @@ impl Builder {
         self
     }
 
+    /// Add a certificate to get passed down when the credentials are acquired.
+    ///
+    /// Certificates passed here may specify a certificate that contains a
+    /// private key to be used in authenticating the application. Typically,
+    /// this is called once for each key exchange method supported by
+    /// servers.
+    ///
+    /// Clients often do not call this function and either depend on Schannel to
+    /// find an appropriate certificate or create a certificate later if needed.
+    pub fn cert(&mut self, cx: CertContext) -> &mut Builder {
+        self.certs.push(cx);
+        self
+    }
+
     /// Creates a new `SchannelCred`.
-    pub fn acquire(&self, direction: Direction) -> io::Result<SchannelCred> {
+    pub fn acquire(&mut self, direction: Direction) -> io::Result<SchannelCred> {
```

Comment:
Looking at the docs, the only thing that might apply here is that you can provide a certificate store with trusted roots for client certificates, but I'm not sure about the ownership semantics of that.